### PR TITLE
Use reflection instead of hardcoding a list of commands

### LINKED ISSUE #119: Use reflection instead of hardcoding a list of commands
Occurrences:

```
HelpWindow.java:140
```

Also remove all the `Command()` constructors.

### PR DESCRIPTION
**Pull request by burnflare**
Fixes #119

### CODE REVIEW

**Comment by burnflare** [comment]
@fanwgwg Please review

File: src/main/java/seedu/agendum/ui/HelpWindow.java [hunk 1]
```diff
@@ -136,18 +133,22 @@ public void handle(KeyEvent evt) {
...
+            try {
+                Map<CommandColumns, String> map = new HashMap<CommandColumns, String>();
+                map.put(CommandColumns.COMMAND, c.getMethod("getName").invoke(null).toString());
+                map.put(CommandColumns.FORMAT, c.getMethod("getFormat").invoke(null).toString());
+                map.put(CommandColumns.DESCRIPTION, c.getMethod("getDescription").invoke(null).toString());
+                commandList.add(map);
+            } catch (Exception e) {
+                logger.severe("Java reflection for Command class failed");
+            }
+        }
```
Comment (burnflare) [inline src/main/java/seedu/agendum/ui/HelpWindow.java:151]: This is the main part

Comment (fanwgwg) [reply src/main/java/seedu/agendum/ui/HelpWindow.java:151]: Yes good! Thanks!

Comment (burnflare) [reply src/main/java/seedu/agendum/ui/HelpWindow.java:151]: Approve the PR?

**Comment by burnflare** [comment]
@fanwgwg Nooooooooo you merged this into the wrong branch. It was based off a non-master branch so that you could view the changes easily. I was going to rebase this off master once the original branch was merged.

**Comment by fanwgwg** [comment]
Oooo it should have been merged into master right???

**Comment by siawyoung** [comment]
Oh my god

**Comment by burnflare** [comment]
@fanwgwg Yes. I'll do the merging, it's ok.

**Pull request merged**